spec/support/config.rb

```diff
@@ -3,9 +3,10 @@
 require 'config'
 
 # NOTE: For some reason `File.expand_path(__dir__, '../..')` did not do the right thing.
-app_root = Pathname.new(__dir__).parent.parent + 'config'
+app_root = Pathname.new(__dir__).parent.parent
```

Comment:
motivated by a complaint from a new cop about preferring interpolation over string concatenation.  this was actually not technically string concatenation, since the returned object is a `Pathname` (`+` is overridden on the `Pathname` class), so a bit of a false positive.  but taking a look at this and trying a switch to `Pathname#join` resulted in slightly clearer var naming while i was tweaking things, so seems like a win.